content/minutes/design-meeting/2019-11-29-repl-meeting.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Read-Eval-Print-Loop (REPL) Extensions
+type: docs
+---
+
+* [Zulip stream][] or read on the [Zulip archive][].
+
+[Zulip stream]: https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/design.20meeting.202019-11-29/near/182183477
+[Zulip archive]: https://zulip-archive.rust-lang.org/131828tcompiler/72407designmeeting20191129.html
+
+# Agenda
+
+- Design doc: https://hackmd.io/GJokfI0wQ0i4SIgRbFTmfw
+
+- Big picture overview and questions
+
+- Essential changes to support REPL extensions
+
+# Big picture overview of design, and on-the-fly questions about it
+
+- Read (to an AST), Compile (AST to MIR), Eval (the MIR to a value), Print (the value). and Loop.
+- Question: Is this going to be a tool analogous to miri, where check-in's that break it may not break CI immediately, but rather will just file follow-up bugs?
+  - Answer 1: it has to be a tool; it depends on miri (or a fork of miri)
+  - Answer 2: it will be even less than a submodule (where a CI automatically files follow-up bugs): There
+    won't be any automatic filing of follow-up bugs (alexreg has said
+    they will handle maintenance manually); this is how miri and
+    clippy had initially started. It could become a swubmodule later.
```

Comment:
```suggestion
    clippy had initially started. It could become a submodule later.
```

Reply:
(I just do transcriptions on behalf of elmer fudd)